packages/css/src/components/gap/README.md

```diff
@@ -9,7 +9,7 @@ Adds white space between a set of elements.
 The five sizes of [Component Space](/docs/brand-design-tokens-space--docs) are available for the width or height of the gap.
 
 | Class name    | Example                                                                                   |
-| ------------- | ----------------------------------------------------------------------------------------- |
+| :------------ | :---------------------------------------------------------------------------------------- |
 | `ams-gap--xs` | <div className="ams-docs-token-example--space" style="inline-size: var(--ams-gap-xs);" /> |
```

Comment:
You didn't use `var(--ams-gap--xs)` here, I think you did intend to?

Reply:
No, these are class names :)

Reply:
Right, my mistake